src/server/graphics/offscreen/display.cpp

```diff
@@ -63,8 +63,11 @@ void mgo::detail::EGLDisplayHandle::initialize()
     if (eglInitialize(egl_display, &major, &minor) == EGL_FALSE)
         BOOST_THROW_EXCEPTION(mg::egl_error("Failed to initialize EGL"));
 
-    if ((major != 1) || (minor != 4))
-        BOOST_THROW_EXCEPTION(std::runtime_error("EGL version 1.4 needed"));
+    if ((major != 1) || (minor < 4))
+        BOOST_THROW_EXCEPTION(
+            std::runtime_error(
+                std::string{"EGL version ≥ 1.4 needed, found "} +
+                std::to_string(major) + "." + std::to_string(minor)));
```

Comment:
Likely a nit, but one day we will see "EGL version ≥ 1.4 needed, found 2.0" which seems misleading or even wrong.